docs/reference/ISM/specify-your-ISM.mdx

```diff
@@ -40,14 +40,22 @@ The Mailbox will call `verify` before delivering a message to its recipient. If
 ```solidity file=<rootDir>/node_modules/@hyperlane-xyz/core/contracts/interfaces/IInterchainSecurityModule.sol#L21
 ```
 
-This is used to signal to the Relayer how to encode `_metadata`. ISMs **must** return one of the supported module types.
+This is used to signal to the Relayer how to encode `_metadata`. ISMs **must** return one of the supported module types:
 
 ```solidity file=<rootDir>/node_modules/@hyperlane-xyz/core/contracts/interfaces/IInterchainSecurityModule.sol#L5-L14
 ```
 
+:::note
+
+To accomplish this, all ISM contracts are initialized with a `moduleType` constant state variable.
```

Comment:
imo "initialized with constant state" is not relevant
but "implements ISM interface and returns moduleType" is